index.js

```diff
@@ -1 +1,220 @@
 console.log("Hello!!!");
+
+//Unique values
+// Check if word in anagrams arr is anagram
+// If you found anagram store this word in set or map
+// Values in set or map should not repeat anagram of this word
+// Also should not be present in set or map
+// Try to optimize script by using check has
+// Return collection set or map
+
+// Example: "actor" and "carot" are anagrams so store only "actor"
+
+const anagrams = [
+  "actor",
+  "carot",
+  "listen",
+  "enlist",
+  "debit",
+  "bidet",
+  "tear",
+  "rate",
+  "night",
+  "thing",
+  "lives",
+  "veils",
+  "stressed",
+  "desserts",
+  "dormitory",
+  "dirty room",
+  "rescue",
+  "secure",
+  "a gentleman",
+  "elegant man",
+  "listen",
+  "heart",
+  "angel",
+  "leader",
+  "silent",
+];
+
+const unique = (arr) => {
+  let x;
+  let y;
+  let z;
+  let litters = new Set();
+  let uniqueAnagrams = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    x = arr[i].split('');
+    y = x.sort();
+    z = y.join("");
+
+    if (!litters.has(z)) {
+      litters.add(z);
+      uniqueAnagrams.push(arr[i]);
+    }
+
+  }
+  return uniqueAnagrams;
+
+};
+
+console.log(unique(anagrams));
+
+
+//Getter and setter
+// Create an user object
+// User should have to first name last name and age Add userInfo getter which gets all info from obj
+// Add userInfo setter to set all user info from string or obj
+// String setter format 'Taras Samoilenko 25'
+
+const user = {
+  firstName: "",
+  lastName: "",
+  age: "",
+  get userInfo(){
+    return `${this.firstName} ${this.lastName} is ${this.age}`
+  },
+  set userInfo(a){
+    if (typeof(a) === 'string') {
+      let arrWords = a.split(" ");
```

Comment:
use const
by the way, you can destruct arrays
const [firstName, lastName, age] = a.split(" ");